tools/c7n_index/c7n_index/metrics.py

```diff
@@ -104,7 +111,17 @@
                     'regions': {'type': 'array', 'items': {'type': 'string'}}
                 }
             }
-        }
+        },
+        'sqs': {
```

Comment:
mixed feelings about this being in the config file vs configurable via cli flag and env var per 12factor style. also region here seems gratitutious given its already a well defined substring in the url.

Reply:
Will refactor to pass sqs info via a cli flag.